great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -695,6 +697,134 @@ def _build_datetime_columns_rule() -> Rule:
+            strict_max=f"{VARIABLES_KEY}strict_max",
+            meta={
+                "profiler_details": {
+                    "column_min_length_range_estimator": f"{column_min_length_range_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}",
+                    "column_max_length_range_estimator": f"{column_max_length_range_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}",
+                },
+            },
+        )
+
+        validation_parameter_builder_configs = [
+            ParameterBuilderConfig(
+                **column_values_to_match_regex_parameter_builder_for_validations.to_json_dict(),
+            ),
+        ]
+        expect_column_values_to_match_regex_expectation_configuration_builder: DefaultExpectationConfigurationBuilder = DefaultExpectationConfigurationBuilder(
+            expectation_type="expect_column_values_to_match_regex",
+            validation_parameter_builder_configs=validation_parameter_builder_configs,
+            condition=f"{column_values_to_match_regex_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}success_ratio >= {VARIABLES_KEY}success_ratio",
+            column=f"{DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}column",
+            regex=f"{column_values_to_match_regex_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY}",
+            mostly=f"{VARIABLES_KEY}mostly",
+            meta={
+                "profiler_details": f"{column_values_to_match_regex_parameter_builder_for_validations.fully_qualified_parameter_name}{FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER}{FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY}",
+            },
+        )
+
+        # Step-5: Instantiate and return "Rule" object, comprised of "variables", "domain_builder", "parameter_builders", and "expectation_configuration_builders" components.
+
+        variables: dict = {
+            "mostly": 1.0,
+            "strict_min": False,
+            "strict_max": False,
+            "false_positive_rate": 0.05,
+            "quantile_statistic_interpolation_method": "auto",
+            "estimator": "bootstrap",
+            "n_resamples": 9999,
+            "random_seed": None,
+            "include_estimator_samples_histogram_in_details": False,
+            "truncate_values": {
+                "lower_bound": 0,
+                "upper_bound": None,
+            },
+            "round_decimals": 0,
+            "success_ratio": 7.5e1,
```

Comment:
@cdkini This needs to be `7.5e-1` -- thanks!

Reply:
done!